src/config/wmodules-github.c

```diff
@@ -9,15 +9,154 @@
+    module->data = github_config;
+
+    // Iterate over module subelements
+
+    for (i = 0; nodes[i]; i++){
+        if (!nodes[i]->element) {
+            merror(XML_ELEMNULL);
+            return OS_INVALID;
+        } else if (!strcmp(nodes[i]->element, XML_ENABLED)) {
+            if (!strcmp(nodes[i]->content, "yes"))
+                github_config->enabled = 1;
+            else if (!strcmp(nodes[i]->content, "no"))
+                github_config->enabled = 0;
+            else {
+                merror("Invalid content for tag '%s' at module '%s'.", XML_ENABLED, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_RUN_ON_START)) {
+            if (!strcmp(nodes[i]->content, "yes"))
+                github_config->run_on_start = 1;
+            else if (!strcmp(nodes[i]->content, "no"))
+                github_config->run_on_start = 0;
+            else {
+                merror("Invalid content for tag '%s' at module '%s'.", XML_RUN_ON_START, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_INTERVAL)) {
+            github_config->interval = time_convert(nodes[i]->content);
+            if (github_config->interval == OS_INVALID) {
+                merror("Invalid content for tag '%s' at module '%s'.", XML_INTERVAL, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_TIME_DELAY)) {
+            github_config->time_delay = time_convert(nodes[i]->content);
+            if (github_config->time_delay == OS_INVALID){
+                merror("Invalid content for tag '%s' at module '%s'.", XML_TIME_DELAY, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_ONLY_FUTURE_EVENTS)) {
+            if (!strcmp(nodes[i]->content, "yes"))
+                github_config->only_future_events = 1;
+            else if (!strcmp(nodes[i]->content, "no"))
+                github_config->only_future_events = 0;
+            else {
+                merror("Invalid content for tag '%s' at module '%s'.", XML_ONLY_FUTURE_EVENTS, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_API_AUTH)) {
+            if (!(children = OS_GetElementsbyNode(xml, nodes[i]))) {
+                continue;
+            }
+            for (j = 0; children[j]; j++){
+                if (!strcmp(children[j]->element, XML_ORG_NAME)) {
+                    if (strlen(children[j]->content) == 0) {
+                        merror("Empty content for tag '%s' at module '%s'.", XML_ORG_NAME, WM_GITHUB_CONTEXT.name);
+                        OS_ClearNode(children);
+                        return OS_INVALID;
+                    }
+                    os_strdup(children[j]->content, github_config->org_name);
```

Comment:
Check if `github_config->org_name` is not NULL. If not NULL, log a warning message indicating that the tag is repeated in the same block.